### ISSUE TITLE
[ANNOUNCEMENT] New Branch

### ISSUE THREAD

**Issue by jcs090218** [jcs-elpa/origami.el#1]
According to the discussion here => https://github.com/melpa/melpa/issues/6716. 

I have started a new branch and looking forward to maintain this package in a more community wise way. If you wish to contribute to this package; please open issue or PR here. I am also looking for people willing to maintain this package together. 👍 

Important notice, you will have to install this branch through [celpa](https://celpa.conao3.com/) runs by @conao3 and not from [melpa](https://melpa.org/). Thanks! 😄

**Comment by jcs090218** [comment]
I am planning to make this fork the official branch from melpa! See https://github.com/melpa/melpa/pull/7215. ;)

**Comment by conao3** [comment]
Wow, good news! :tada:

**Comment by sandinmyjoints** [comment]
From the discussion that thread, seems like it is not going to end up on melpa? I just installed from celpa. Eager to try it out!

**Comment by jcs090218** [comment]
Yes, please use [celpa]() for this branch! Feel free to open bug report or feature request!

**Comment by emacksnotes** [comment]
I don't want to get in to politics.  

I had opened the following an issue [https://github.com/gregsexton/origami.el/issues/86](Backtrace when M-x origami-recursively-toggle-node on xdisp.c@Emacs 26.2 ) in the parent repo.  It is clearly some issue with the parser, and I would consider it a core issue.  Unfortunately, that issue remains unaddressed for almost 2 years.

I am happy that this fork exists, and I would like to contribute some improvements.

As for putting this repo in melpa, have you considered re-naming that package to `origami+`, or `origami-version-number+` or some such thing.

**Comment by emacksnotes** [comment]
It is possible that original author of `origami` has a signed assignment with FSF, and plans to put his project on GNU ELPA.  If this is the case,  there could be reasons why he might want to restrict the contributors.  If he accepts contribution from every tom, dick and harry, then Emacs contributors are going to ask him to purify his code. I know of packages which would very much want to be in GNU ELPA, but stops short because purging contributions of non-FSF contributors is not particularly a enjoyable task.  

Just my 2 cents.

**Comment by jcs090218** [comment]
> I had opened the following issue Backtrace when M-x origami-recursively-toggle-node on xdisp.c@Emacs 26.2 in the parent repo. It is clearly some issue with the parser, and I would consider it a core issue. Unfortunately, that issue remains unaddressed for almost 2 years.
> 
> I am happy that this fork exists, and I would like to contribute some improvements.

That's great to hear! This is exactly why I have this fork. Feel free to contribute to this repository.

> As for putting this repo in to melpa, have you considered re-naming that package to origami+, or origami-version-number+ or some such thing.

I think we can rename it to `origami+`.

> It is possible that original author of origami has a signed assignment with FSF, and plans to put his project on GNU ELPA. If this is the case, there could be reasons why he might want to restrict the contributors. If he accepts contribution from every tom, dick and harry, then Emacs contributors are going to ask him to purify his code. I know of packages which would very much want to be in GNU ELPA (for example, ESS, BBDB, use-packages etc), but stops short because purging contributions of non-FSF-assigned authors from the codebase is not an enjoyable task for anyone.

Thanks for the information. I don't know the original author of origami @gregsexton so I don't know if he is planning to push his package to GNU ELPA. If he already signed assignment with FSF then we can certainly consider pushing this package to GNU ELPA.
